eclass/ruby-ng.eclass

```diff
@@ -183,11 +183,7 @@ _ruby_wrap_conditions() {
 	local conditions="$1"
 	local atoms="$2"
 
-	for condition in $conditions; do
-		atoms="${condition}? ( ${atoms} )"
-	done
-
-	echo "$atoms"
+	echo "${conditions[@]/#/? ( ${atoms} )/}"
```

Comment:
This doesn't seem to do the same thing:

```
# before
$ _ruby_wrap_conditions '1 2 3' 'a b c'
3? ( 2? ( 1? ( a b c ) ) )
# after
? ( a b c )/1 2 3
```

Reply:
Yeah, this isn't equivalent - I'd misread what it was doing originally.